internal/controllers/import_controller.go

```diff
@@ -257,7 +258,7 @@ func (r *CAPIImportReconciler) reconcileNormal(ctx context.Context, capiCluster
 		Namespace: capiCluster.Namespace,
 	}
 
-	remoteClient, err := r.remoteClientGetter(ctx, capiCluster.Name, r.Client, clusterKey)
+	remoteClient, err := r.remoteClientGetter(ctx, capiCluster.Name, r.RancherClient, clusterKey)
```

Comment:
If the providers are installed outside of rancher management cluster, then likely the secret to connect to the CAPI cluster will live inside this “other then rancher manager” cluster. So here for me it makes sense to use regular in-cluster client to connect to the provisioned cluster by CAPI from the cluster where CAPI is installed.